<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Baraban</class>
 <widget class="QDialog" name="Baraban">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Fonlabel">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>-5</y>
     <width>811</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BarabanObjectLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>457</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Крутить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton100">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>457</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton500">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>457</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1000">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>457</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="krutka1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="krutka2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="krutka3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_stavka">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>22</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ставка: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_robux">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="count_robux">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
